src/core/client/stream/tabs/Comments/Comment/CommentContainer.css

```diff
@@ -76,3 +76,7 @@ $commenterActionEditColorActive: var(--palette-primary-300);
 .notSeen {
   background-color: $colors-yellow-100;
 }
+
+.flattenedPadding {
+  padding-left: var(--spacing-4);
```

Comment:
`spacing-4` is too little and `spacing-5` too big... feel free to set this to `18px` instead :-D